Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -613,14 +613,25 @@ inline TextAlignMode BuilderConverter::convertTextAlign(BuilderState& builderSta
     auto& primitiveValue = downcast<CSSPrimitiveValue>(value);
     ASSERT(primitiveValue.isValueID());
 
+    const auto& parentStyle = builderState.parentStyle();
+
+    // User agents are expected to have a rule in their user agent stylesheet that matches th elements that have a parent
+    // node whose computed value for the 'text-align' property is its initial value, whose declaration block consists of
+    // just a single declaration that sets the 'text-align' property to the value 'center'.
+    // https://html.spec.whatwg.org/multipage/rendering.html#rendering
+    if (primitiveValue.valueID() == CSSValueInternalThCenter) {
+        if (parentStyle.textAlign() != TextAlignMode::Start)
```

Comment:
```suggestion
        if (parentStyle.textAlign() != RenderStyle::initialTextAlign())
```

The spec mentions initial value, so this should be more explicit.

Reply:
Yes. Also going to swap the conditional to better match the text from the spec.